src/components/CandidateForm.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import styles from "./CandidateForm.module.css";
+import Button from "./Button";
+
+
+const CanditateForm = ({inputsData, sendFormData}) => {
+    const inputs = inputsData;
+
+    const getFormData = () => {
+      const formData = {};
+      document.querySelectorAll('input').forEach((input) => {
```

Comment:
This is not the React way of selecting and manipulation DOM elements. What you should to do here is to pass an onChange event to the input element. That event will trigger a function every time you type something into the input. You should pass an argument to that function (usually is 'e', or 'event') and that argument will have access to the element. This way you can access the attributes or the value of that element eg. `e.target.name`, ` e.target.value` .